### PR TITLE
demo pr

### PR DESCRIPTION
**Pull request by ming1in**
this is a sick description

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -22,7 +22,13 @@ import {
...
 } from "@octokit/webhooks-types";
 import { minBy, sortBy, flatten } from "lodash";
-import { addInitialComment, addComment, getApproveReview, postReviewComentReply, getReviewComments } from "./github";
+import {
+  addInitialComment,
+  addComment,
+  getApproveReview,
+  postReviewComentReply,
+  getReviewComments,
+} from "./github";
```
Comment (coreyja) [inline app.ts:31]: This is a good import!

Comment (coreyja) [reply app.ts:31]: We did domain-ish files!

File: app.ts [hunk 2]
```diff
@@ -60,15 +66,17 @@ const githubApp = new GithubApp({
   privateKey: process.env.GITHUB_PRIVATE_KEY,
 });
 
-const channelNameFromPull = (pull: Pick<PullRequest, 'number' | 'base'>): string => `pr-${pull.number}-${pull.base.repo.name}`
+const channelNameFromPull = (
```
Comment (JackRegueiro) [inline app.ts:69]: Love this!

**Review by fredlinde — APPROVED** [review]
Awesome!

**Review by aalobai1 — APPROVED** [review]
lgtm

File: app.ts (hunk 1 above)
Comment (gnaratil2017) [reply app.ts:31]: continue the thread

Comment (stevenwchien) [reply app.ts:31]: I agree

File: app.ts [hunk 3]
```diff
@@ -22,7 +22,13 @@ import {
   PullRequestReviewSubmittedEvent,
 } from "@octokit/webhooks-types";
 import { minBy, sortBy, flatten } from "lodash";
-import { addInitialComment, addComment, getApproveReview, postReviewComentReply, getReviewComments } from "./github";
+import {
```
Comment (satub) [inline app.ts:25]: 👀

File: app.ts (hunk 2 above)
Comment (ming1in) [reply app.ts:69]: yeah i love this too

Comment (ming1in) [reply app.ts:69]: nit: this is good

Comment (gnaratil2017) [reply app.ts:69]: this is great!

Comment (coreyja) [reply app.ts:69]: I'm not sure I do! Lets take it to Slack 🚀

Comment (coreyja) [reply app.ts:69]: Lets take this to slack!

File: app.ts [hunk 4]
```diff
@@ -60,15 +66,17 @@ const githubApp = new GithubApp({
   privateKey: process.env.GITHUB_PRIVATE_KEY,
 });
 
-const channelNameFromPull = (pull: Pick<PullRequest, 'number' | 'base'>): string => `pr-${pull.number}-${pull.base.repo.name}`
+const channelNameFromPull = (
+  pull: Pick<PullRequest, "number" | "base">
```
Comment (satub) [inline app.ts:70]: Pick or NitPick? 🤔

**Pull request closed without merging**